test/images/port-forward-tester/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM scratch
+ADD portforwardtester portforwardtester
+ADD portforwardtester.go portforwardtester.go
```

Comment:
I probably wouldn't otherwise but I recently read this comment: https://github.com/kubernetes/kubernetes/pull/13982/files#r39559798